connector/saml/saml.go

```diff
@@ -366,6 +366,8 @@ func (p *provider) HandlePOST(s connector.Scopes, samlResponse, inResponseTo str
 		return ident, fmt.Errorf("response did not contain a AttributeStatement")
 	}
 
+	p.logger.Infof("parsed and verified saml response attributes %s", attributes)
```

Comment:
The intent here is to spit out all the attributes that the server actually sent us. Helps us identify "you didn't configure your SAML provider to send us the correct set of information."

If the admin has done the correct mapping the attributes will show ups in the "login successful:" entry you commented on above anyway.